addons/mass_mailing/models/mailing.py

```diff
@@ -518,12 +518,17 @@ def _action_view_traces_filtered(self, view_filter):
 
     def action_view_clicked(self):
         model_name = self.env['ir.model']._get('link.tracker').display_name
+        helper = (
```

Comment:
Just for clarity, I would define helper_header and helper_message, and use them below :) code would be easier to follow that way